### DISCUSSION TITLE
Kanboard 1.2.29 not work anymore with mariaDB

### DISCUSSION THREAD

**Discussion by mariushosting** [kanboard/kanboard#5280] [Installation & Configuration]
Kanboard 1.2.29 is not work anymore with mariaDB. Only Kanboard 1.2.28 is working good.
There is some breaking change on the 1.2.29 version that break thousand of DB installations.

Internal Error: Unable to run SQL migrations: Running migration \Schema\version_45, SQLSTATE[42S21]: Column already exists: 1060 Duplicate column name 'date_moved' (You may have to fix it manually)

**Comment by goaen** [comment]
I had the same error, but got around it by utilizing mariaDB syntax of "ADD COLUMN IF NOT EXISTS" and "CREATE TABLE IF NOT EXISTS" in Mysql.php. I just started looking into Kanboard, so I'm not sure if there are any side effects of this change though...(?)

**Reply by rbouqueau** [reply]
Was this solved? My hoster updated their MariaDB and I now have this issue. I want to check before updating Kanboard. Thank you!
